### PR TITLE
Update loadEBSD_h5oina.m

### PR DESCRIPTION
**Pull request by vtvivian**
import images as ebsd.opt.Images.(name)

### CODE REVIEW

File: interfaces/loadEBSD_h5oina.m [hunk 1]
```diff
@@ -89,7 +135,7 @@
         %read all datsets
         for est=1:length(allEDS)
             for thing = 1:length(allEDS{est})
-                sane_name = regexprep(allEDS{est}(thing).Name,{' |-|,|:|%|~|#' char(945) char(946)},{'_' 'a' 'b'});
+                sane_name = regexprep(allEDS{est}(thing).Name,' |-|,|:|%|~|#','_');
```
Comment (ralfHielscher) [inline interfaces/loadEBSD_h5oina.m:138]: Hi Vivian,
can you comment, why you did this change?
Ralf

File: interfaces/tools/calcUnitCell.m [hunk 2]
```diff
@@ -36,7 +36,7 @@
 end
 
 % reduce data set
-if length(xy)>10000, xySmall = subSample(xy,10000); end
+if length(xy)>10000, xySmall = subSample(xy,10000); else xySmall=xy; end
```
Comment (ralfHielscher) [inline interfaces/tools/calcUnitCell.m:39]: This is an important bug fix. I made this a commit for it's self. 
Thank you for pointing this out.

**Comment by ralfHielscher** [comment]
Hi Vivian,
thank you for your work. Can you provide me with a data set that contains all the fields that you import with the new interface?
Ralf.

File: interfaces/loadEBSD_h5oina.m (hunk 1 above)
Comment (vtvivian) [reply interfaces/loadEBSD_h5oina.m]: sorry, this was an accidental change when I tried to merge mtex-toolbox/develop into my fork! I reverted it in the next commit.

**Comment by ralfHielscher** [comment]
Thank you Vivian for your contribution.

Ralf.

**Pull request merged**